chain/types/message.go

```diff
@@ -21,18 +21,18 @@ type Message struct {
 	// Address of the sending actor.
 	From address.Address
 	// Expected CallSeqNum of the sending actor (only for top-level messages).
-	CallSeqNum int64
+	CallSeqNum uint64
 
 	// Amount of value to transfer from sender's to receiver's balance.
 	Value big.Int
 
+	GasPrice big.Int
+	GasLimit int64
+
 	// Optional method to invoke on receiver, zero for a plain value send.
 	Method abi.MethodNum
 	/// Serialized parameters to the method (if method is non-zero).
 	Params []byte
-
-	GasPrice big.Int
-	GasLimit int64
 }
 
 func (t *Message) MarshalCBOR(w io.Writer) error {
```

Comment:
I copied these marshalers from lotus as it was easier than writing the code gen here.

Reply:
Please write something like this in the code, not just this PR.